client/src/pages/Conversation/Conversations.tsx

```diff
@@ -0,0 +1,43 @@
+import React, { useEffect } from 'react';
+import { Grid, Typography } from '@material-ui/core';
+import ConversationSideBar from '../../components/Conversation/ConversationSideBar/ConversationSideBar';
+import ConversationChat from '../../components/Conversation/ConversationChat/ConversationChat';
+import { useConversation } from '../../context/useConversationContext';
+import { useSocket } from '../../context/useSocketContext';
+import { useAuth } from '../../context/useAuthContext';
+import useStyles from './useStyles';
+const Conversations = (): JSX.Element => {
+  const classes = useStyles();
+  const { currentConversation } = useConversation();
+  const { loggedInUser } = useAuth();
+  const { socket } = useSocket();
+  useEffect(() => {
+    if (socket) {
+      if (!currentConversation || !loggedInUser) return;
+      socket.emit('JoinConversation', { userId: loggedInUser._id, conversationId: currentConversation._id }, () => {
```

Comment:
```suggestion
      socket.emit('joinConversation', { userId: loggedInUser._id, conversationId: currentConversation._id }, () => {
```

JavaScript convention is camel case, there isn't really a need for making this PascalCase